src/ppx_deriving_yojson_runtime.cppo.ml

```diff
@@ -11,7 +11,11 @@ let rec map_bind f acc xs =
   | x :: xs -> f x >>= fun x -> map_bind f (x :: acc) xs
   | [] -> Result.Ok (List.rev acc)
 
+#if OCAML_VERSION >= (4, 08, 0)
+type 'a error_or = ('a, string) Result.t
+#else
```

Comment:
This should be unnecessary now that https://github.com/ocaml-ppx/ppx_deriving/pull/197 is merged.

Reply:
I think this is still necessary, as https://github.com/ocaml-ppx/ppx_deriving/pull/197 only added `Result.result` to the branch where `OCAML_VERSION` is less than 4.08

Reply:
The alternative is to move `type 'a error_or = ('a, string) Result.t` to the top of the file so that it appears before the `include Ppx_deriving_runtime` and sees the unshadowed (at the time) compatibility `Result` module (from https://github.com/janestreet/result)